### ISSUE TITLE
EPUB 3 references in the A11y documents

### ISSUE THREAD

**Issue by iherman** [w3c/epub-specs#1552]
Both the a11y and the a11y-techiques document refer to:

```
EPUB 3. W3C. URL: https://www.w3.org/publishing/epub3/
```

I believe this should be changed for the EPUB 3.3 references now...

**Comment by mattgarrish** [comment]
Looks like TR/epub redirects to the latest spec now, so that's what I believe we needed.

**Comment by iherman** [comment]
> Looks like TR/epub redirects to the latest spec now, so that's what I believe we needed.

Yes. Just use the undated URI
